packages/cli/src/commands/init/init.ts

```diff
@@ -171,12 +171,22 @@ async function installDependencies({
   loader.succeed();
 }
 
+function validateOption(optName: string, optValue: string) {
+  if (typeof optValue === 'boolean') {
+    logger.error(`error: '${optName} <string>' argument missing`);
+    process.exit(1);
```

Comment:
I would prefer we `throw new CLIError` here to stay consitent with other parts of the codebase.